plutus-pab/src/Plutus/PAB/Core/ContractInstance/STM.hs

```diff
@@ -424,13 +412,6 @@ waitForTxStatusChange oldStatus tx BlockchainEnv{beTxChanges} = do
             guard $ oldStatus /= newStatus
             pure newStatus
 
--- | The value at an address
-valueAt :: Address -> BlockchainEnv -> STM Value.Value
-valueAt addr BlockchainEnv{beAddressMap} = do
-    am <- STM.readTVar beAddressMap
-    let utxos = view (AM.fundsAt addr) am
-    return $ foldMap (txOutValue . txOutTxOut) utxos
-
```

Comment:
Moved to Plutus.PAB.Core, and changed the implementation to query the chain index.